eng/ci/templates/variables/crank.yml

```
@@ -0,0 +1,12 @@
+variables:
+- name: buildId
+  value: $(Build.BuildId)
+- name: buildNumber
+  value: $(Build.BuildNumber)
+- name: sourceVersion
+  value: $(Build.SourceVersion)
+- name: storeBenchmarkResultsInDatabase
+  ${{ if eq(variables['Build.Reason'], 'Schedule') }}:
```

Can remove the if/esle
```suggestion
  value: ${{ eq(variables['Build.Reason'], 'Schedule') }}
```

---

Changed to suggested.